jwst/assign_wcs/util.py

```diff
@@ -807,6 +807,56 @@ def update_s_region_imaging(model):
         model.meta.wcsinfo.s_region = s_region
 
 
+def update_s_region_lrs(model, reference_files):
+    """
+    Determine ``S_REGION`` using V2,V3 of the slit corners from reference file.
+
+    Parameters
+    ----------
+    model : DataModel
+        Input model
+    reference_files : list
+        List of reference files for assign_wcs.
+
+    Returns
+    -------
+    None
+        s_region for model is updated in place .
+
```

Comment:
```suggestion
```

Reply:
if you think it's useful to specify the update is done in-place, it can be put in the "extended summary"